config.go

```diff
@@ -48,16 +48,22 @@ const (
 	ccProxy = "cc"
 )
 
+const (
+	ccShim = "cc"
```

Comment:
If this is the name of the config file section for the shim, wouldn't it be better to call it `shim`?